### ISSUE TITLE
podman cni error on k8s host with calico cni running

### ISSUE THREAD

**Issue by sdeoras** [containers/libpod#3874]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

I have a k8s 1.15.2 cluster with Ubuntu 18.04 based nodes with CRI-O and Calico CNI. Running podman errors out as shown below. I would like to know if there is a way to configure system such that both k8s CNI and podman CNI can co-exist.

**Steps to reproduce the issue:**

1. Setup k8s 1.15.2 with Calico networking

2. Run podman on alpine image

3.

**Describe the results you received:**
```
sudo podman run --rm docker.io/library/alpine:latest date
ERRO[0000] Error adding network: incompatible CNI versions; config is "0.4.0", plugin supports ["0.1.0" "0.2.0" "0.3.0" "0.3.1"] 
ERRO[0000] Error while adding pod to CNI network "podman": incompatible CNI versions; config is "0.4.0", plugin supports ["0.1.0" "0.2.0" "0.3.0" "0.3.1"] 
Error: error configuring network namespace for container a6167fabfb7f861e5d3e0682384963c2ef14e8f69530bfc439a80f6b34f5c29f: incompatible CNI versions; config is "0.4.0", plugin supports ["0.1.0" "0.2.0" "0.3.0" "0.3.1"]
```

**Describe the results you expected:**
Expected it to pick default cni configuration and run

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**
```
podman version
Version:            1.5.0
RemoteAPI Version:  1
Go Version:         go1.10.4
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
podman info --debug
debug:
  compiler: gc
  git commit: ""
  go version: go1.10.4
  podman version: 1.5.0
host:
  BuildahVersion: 1.10.1
  Conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.1-dev, commit: 8a56404fa7759f4901cf989e60d31c1e04af53db'
  Distribution:
    distribution: ubuntu
    version: "18.04"
  MemFree: 283336704
  MemTotal: 7836004352
  OCIRuntime:
    package: Unknown
    path: /usr/bin/runc
    version: |-
      runc version 1.0.0-rc6
      spec: 1.0.1-dev
  SwapFree: 0
  SwapTotal: 0
  arch: amd64
  cpus: 2
  eventlogger: journald
  hostname: ubuntu-mbsb
  kernel: 4.15.0-1032-gcp
  os: linux
  rootless: true
  uptime: 1h 58m 0.63s (Approximately 0.04 days)
registries:
  blocked: null
  insecure: null
  search:
  - registry.access.redhat.com
  - registry.fedoraproject.org
  - quay.io
  - docker.io
store:
  ConfigFile: /home/sdeoras/.config/containers/storage.conf
  ContainerStore:
    number: 0
  GraphDriverName: vfs
  GraphOptions: null
  GraphRoot: /home/sdeoras/.local/share/containers/storage
  GraphStatus: {}
  ImageStore:
    number: 0
  RunRoot: /run/user/1002
  VolumePath: /home/sdeoras/.local/share/containers/storage/volumes

```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/bionic,now 1.5.0-2~ubuntu18.04~ppa5 amd64 [installed]
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by mheon** [comment]
I believe @baude is working on the final stages of updating Podman to support that version of the plugins - so Podman 1.5.2 should have support

**Comment by mheon** [comment]
I believe this is fixed upstream by a revendor of `ocicni` in 1.5.2 - please reopen if it's not.
